scripts/install/opt/seagate/cortx/motr/libexec/motr_cfg.sh

```diff
@@ -201,6 +202,8 @@ lvm_size_percentage_diff()
 
 motr_cfg_update()
 {
+    MOTR_CONF_FILE = $1
```

Comment:
This will break R2 way of deployment on hardware, if conf file is not passed do_m0provision_action should still succeed because motr_cfg_update expects an argument, MOTR_CONF_FILE is not set, the code will fail to execute the loop. need to set the MOR_CONF_FILE as an argument
options:
1. always pass conf file as argument.
2. [ -n $MOTR_CONF_FILE ] && MOTR_CONF_FILE = $1